frontend/assets/l10n/app_de.json

```diff
@@ -22,6 +22,16 @@
     "sourceCode": "Quellcode der App",
     "title": "Über"
   },
+  "activation" : {
+    "headline": "Karte aktivieren",
+    "description": "Wollen sie diese Karte aktivieren?",
+    "limitReached": "Ihr Kartenlimit wurde erreicht. Bitte entfernen sie vorher eine Karte, damit sie eine neue aktivieren können.",
+    "buttonText": "Karte aktivieren",
+    "invalidCode": "Ihr Aktivierungscode ist ungültig.",
+    "alreadyExists": "Diese Karte wurde bereits auf dem Gerät aktiviert.",
+    "activationSuccessful": "Die Aktivierung war erfolgreich.",
+    "activationInvalid": "Die Aktivierung befindet sich in einem ungültigen Zustand."
+  },
```

Comment:
Shouldn't these be inside the `identification` group of keys?

Reply:
If not, maybe rename the group to sth like `deeplinkActivation` or so to make it clear that these keys belong only to the deeplink activation workflow

Reply:
no this is a separate route but i can rename it